patterns/2-structured/core-team.md

```diff
@@ -43,8 +45,7 @@ It's clearly due for an overhaul, but even though everyone needs that and wants
 
 ## Solution
 
-Form a core team whose job it is to maintain the project in a state so that others can easily onboard and contribute to it.
-This core team does the work that is necessary for a healthy usage and contribution ecosystem.
+Form a core team whose job it is to maintain this project in a state so that others can easily onboard and contribute to it.This core team does the work that is necessary for a healthy usage and contribution ecosystem.
```

Comment:
```suggestion
Form a core team whose job it is to maintain this project in a state so that others can easily onboard and contribute to it.
This core team does the work that is necessary for a healthy usage and contribution ecosystem.
```

@rrrutledge I suspect that this was an accident, as you got one sentence per row everywhere else?